src/components/inversion_component/inversion.sh

```diff
@@ -14,19 +14,23 @@ setup_inversion() {
     cd ${OutputPath}/$RunName
     mkdir -p -v inversion
     mkdir -p inversion/data_converted
+    mkdir -p inversion/data_converted_background
     mkdir -p inversion/data_geoschem
+    mkdir -p inversion/data_geoschem_background
```

Comment:
Do we want to create `inversion/data_converted_background` and `inversion/data_geoschem_background` even when `LognormalErrors: false` ?

Reply:
I think users would generally want to create a new inversion directory when switching from normal to lognormal errors (for example as part of an inversion ensemble)

Reply:
I initially did this to make it easier to test normal vs lognormal since the precomputed  jacobian option is currently broken when not using KF mode. I think you are right that we can just use a single directory for both.

One question is that under the current implementation if one ran a normal inversion and then wanted to do a lognormal inversion they would need to manually create and run a background simulation, as that simulation is not used in the normal inversion, but perhaps we should have it run that additional simulation by default, even in the normal case, to prevent the need for manual generation? Starting lognormal and switching to normal should work fine as is.

Reply:
Another option that would be a bit more work but maybe more user friendly is to add a logical along these lines:

```
if $LognormalErrors is true and $PrecomputedJacobian is true:
    if the background simulation has not been run:
        run the background simulation
```

That way users running normal inversions wouldn't need to run an extra unnecessary simulation.

What do you think?